src/Symfony/Component/VarDumper/Tests/Caster/DateCasterTest.php

```diff
@@ -30,10 +30,10 @@ class DateCasterTest extends TestCase
      */
     public function testDumpDateTime($time, $timezone, $xDate, $xTimestamp)
     {
-        $date = new \DateTime($time, new \DateTimeZone($timezone));
+        $date = new \DateTimeImmutable($time, new \DateTimeZone($timezone));
```

Comment:
This tests should be preserved. We still want to cover the dumping of a DateTime to make sure we support it. We should have separate tests covering DateTimeImmutable instead.

Reply:
Ok thanks @stof 😁